bundles/azure/aso/psql/bundle/config/01-flexible-server.ytt.yml

```diff
@@ -0,0 +1,229 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:regexp", "regexp")
+#@ load("@ytt:ip", "ip")
+
+
+#@ flexibleServerName    = data.values.resource_name
```

Comment:
nit: I think we want a check on `resource_name` for it to be not `""`

Reply:
Yep, will do.